src/components/Toolbar/ToolbarConstants.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
can we rename this file to simply `constants.js` to match the other components?

Reply:
I actually find it confusing to have just `constants.js` which contain component specific constants.

Reply:
ok... no issues there then. I guess this is a bit more specific (as they are only specific to the Toolbar and not other sub-toolbar components). I guess typically I have been including those in the component itself in those cases, but fine with this convention. Is this the correct understanding?

Reply:
I just like to have the file named for what it is so if I have multiple open in my IDE I can see which is the Toolbar constants file vs the Button constants file.

Reply:
One could say, like a "constant" reminder...

Reply:
I would agree with this. So if it's the global change, I think we leave it `ToolbarConstants.js` and then apply the same to others in a separate PR... I just wanted to make sure I understood ;)